Ryujinx.HLE/HOS/Services/Hid/Types/SharedMem/Npad/Npad.cs

```diff
@@ -0,0 +1,21 @@
+namespace Ryujinx.HLE.HOS.Services.Hid
+{
+    // TODO: Add missing structs
+    unsafe struct ShMemNpad
+    {
+        public NpadStateHeader Header;
+        public Array7<NpadLayout> Layouts; // One for each NpadLayoutsIndex
+        public Array6<NpadSixAxis> Sixaxis;
+        public DeviceType DeviceType;
+        uint padding1;
+        public NpadSystemProperties SystemProperties;
+        public uint NpadSystemButtonProperties;
+        public Array3<BatteryCharge> BatteryState;
+        fixed byte NfcXcdDeviceHandleHeader[0x20];
+        fixed byte NfcXcdDeviceHandleState[0x20 * 2];
+        ulong mutex;
```

Comment:
```suggestion
        ulong Mutex;
```
(Only if you made is public)